tests/test_cookiecutter_local_no_input.py

```diff
@@ -65,7 +66,7 @@ def test_cookiecutter_no_input_return_rendered_file():
     """Verify Jinja2 templating correctly works in `cookiecutter.json` file."""
     project_dir = main.cookiecutter('tests/fake-repo-pre', no_input=True)
     assert project_dir == os.path.abspath('fake-project')
-    with open(os.path.join(project_dir, 'README.rst')) as fh:
+    with Path(project_dir, 'README.rst').open() as fh:
         contents = fh.read()
```

Comment:
```suggestion
    contents = Path(project_dir, 'README.rst').read_text()
```
I haven't looked at all the changes, but I suspect this could be used in many places?

Reply:
good idea. Will do, anyway I need to return encoding

Reply:
in last updates.